### ISSUE TITLE
Experimental\Electric fix

### ISSUE THREAD

**Issue by cprotopa** [open-ideas/IDEAS#673]
Most models in this package are not working properly, because they haven't been updated with IDEAS / Modelica changes.

**Comment by Mathadon** [comment]
Are there some examples that you suggest to add a unit test to? This could help to avoid future bugs.

**Comment by cprotopa** [comment]
I have further cleaned up the package and "everything" seems to work, so far as I can judge. I made a pull request #725. 

I'm not familiar with all models, but there are examples in the different sub-packages that could perhaps be used for unit tests. For instance: 
```
IDEAS.Experimental.Electric.Batteries.Examples.TestBatteryAC
IDEAS.Experimental.Electric.Distribution.Examples.TestGridGeneral
IDEAS.Experimental.Electric.Photovoltaics.Examples.UsePVGeneral
```
However, for the last, an input file is necessary, non existant for the moment.
How do those unit-tests work? Do you need to define specific conditions and simulation settings?

**Comment by Mathadon** [comment]
@cprotopa that's great, thanks!

The unit test scripts are in `IDEAS/Resources/Scripts`. There are plenty of examples there. The bottom line is that it must create a simulate command and plot commands for the variables that should be stored in the unit test. If you add scripts then I can run them for you.

**Comment by Mathadon** [comment]
closed through #725
